serialize.py

```
@@ -0,0 +1,19 @@
+import pickle
+
+def deserialize_data(data_file):
+    '''
+    Reads serialized data from a given file
+    '''
+    try:
+        with open(data_file, 'rb+') as f:
+            data = pickle.load(f)
+    except:
```

Using `except:` with no defined catches is bad practice, as it'll silence _all_ errors that may happen.  A better idea is specific except blocks for the exact errors you plan to catch and handle, that way any other unexpected errors can bubble up providing their trace for help in debugging instead of just silently continuing.